server_static/documentazione/programmazione_annuale.md

```diff
@@ -1,2 +1,51 @@
-## programmazione annuale
+<style>
+    @import url(/css/doc-style.css);
+</style>
 
+## TEMPLATE DOCENTE
+
+La schermata template docente della sezione *PROGRAMMAZIONE ANNUALE* mostra una tabella con varie colonne contenenti la lista di tutte le programmazioni fatte fino a quel momento. In alto a destra vi è un <span class="button blue">bottone</span> per creare nuovi template.
+
+![tabella_programmazioneannuale](/img/documentazione/programmazione-annuale/prova1.png)
+
+
+Contenuti delle colonne:
+
+- **NOME**: nome della programmazione
+- **MATERIA**: nome della materia
+- **AZIONI**: per ogni programmazione sono presenti 3 azioni: aggiorna, duplica e elimina
+  - **AGGIORNA**: cliccando sull'icona è possibile modificare il template selezionato. Una volta aggiornato, bisogna cliccare il pulsante "Aggiorna template" oppure "Cancel" per cancellare le modifiche.
+  - **DUPLICA & CONDIVIDI**: cliccando sull'icona è possibile inviare e condividere il template selezionato. Bisognerà selezionare il docente a cui si vuole inviare nel menù a tendina.
+  - **ELIMINA**: cliccando sull'icona è possibile eliminare il template.
+
+
+## nuovo template
+
+![tabella_programmazioneannuale](/img/documentazione/programmazione-annuale/prova2.png)
+
+- **NOME TEMPLATE**: nome del template
+- **MATERIA**: nome della materia
+- **LIBRO DI TESTO**: nome del libro di testo, bisogna aggiungere tutti i parametri richiesti.
+- **NOTE**: aggiunta di note della programmazione
+- **TRIMESTRE**: nella sezione trimestre si devono aggiungere tutti gli argomenti del trimestre
+- **PENTAMESTRE**: nella sezione pentamestre si devono aggiungere tutti gli argomenti del pentamestre
+
+## PROGRAMMAZIONE DOCENTE
```

Comment:
spiegare cosa vede un docente e cioè che vede solo le materie per le classi in cui insegna